studies/forms.py

```diff
@@ -329,7 +329,6 @@ class StudyEditForm(StudyForm):
     def __init__(self, user=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields["external"].disabled = True
-        self.fields["scheduled"].disabled = True
```

Comment:
Does `StudyEditForm` control the form's initial settings when a study is first created, or when the Edit button is clicked after the study has been created, or both? I'm guessing it's the latter, and thus the External checkbox still needs to be disabled because it was set when the study was created and can't change after that. Is that right?

Reply:
I would disagree.  I think tests should go with the PR.  I didn't include a PR, because I felt that this was a UI change that can't be tested with our current testing framework.  But you're correct, this is a change with the form class and should have a test to go with this change.

Reply:
> I would disagree. I think tests should go with the PR. 

Ok, good to know! 😃 

> I felt that this was a UI change that can't be tested with our current testing framework

Ah ok, I thought we could test UI changes - in that case ignore me (or add a form class test if that's relevant/possible).